share/picongpu/tests/FieldAbsorber/etc/picongpu/4.cfg

```diff
@@ -39,10 +39,12 @@ TBG_devices_z=1
 
 # When changing the number of cells consider changing positionX, positionY in FieldBackgroundJ::operator().
 # To match the setup from the Taflove-Hagness book, the size should be 40 + PML size min border + PML size max border.
-TBG_numCells=60
+TBG_numCells=1060
```

Comment:
number does not match setup in ci but the documentation 
please fix

Reply:
Done